src/services/user.service.ts

```diff
@@ -74,3 +74,34 @@ export async function deleteUser(id: number) {
   await user.remove();
   return user;
 }
+
+const RANDOM_TERM = [
+  '알파카',
+  '너구리',
+  '감자도리',
+  '뱁새',
+  '개구리',
+  '오리너구리',
+  '오이',
+  '빛과소금',
+  '복숭아',
+  '튤립',
+];
+
+const randomNumber = (min, max) =>
+  Math.floor(Math.random() * (max - min + 1) + min);
+
+export async function getRandomNickname() {
+  const randomNickname = `${RANDOM_TERM[randomNumber(0, 9)]}${randomNumber(
+    1,
+    100000
+  )}`;
```

Comment:
`RANDOM_TERM` 의 크기를 매직 넘버 말고 `length` 를 이용해보는 건 어떨까요?

```suggestion
  const lengthOfTerm = RANDOM_TERM.length;
  const randomTerm = `${RANDOM_TERM[randomNumber(0, lengthOfTerm - 1]}`;
  const randomNumber = randomNumber(1, 100000);
  const randomNickname = `${randomTerm}${randomNumber}`;
```

Reply:
그러게요 말씀해주신 방법이 좋을 것 같습니다!